app/controllers/posts_controller.rb

```diff
@@ -3,7 +3,26 @@ def index
     @user = User.find(request.parameters[:user_id])
   end
 
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
+    if @post.save
+      redirect_to start_path
+    else
+      puts @post.errors.full_messages
+    end
+  end
+
   def show
     @post = Post.find(request.parameters[:id])
   end
+
+  private
+
+  def post_params
+    params.require(:post).permit(:author, :title, :text)
+  end
```

Comment:
- [ ] kindly fix this error preventing users from creating a post 🙏  : 

![image](https://user-images.githubusercontent.com/89282221/218085779-46cb7168-e50e-4deb-b3bc-c73c12f1d103.png)